lib/src/main/java/vmath/Mat3.java

```diff
@@ -10,4 +10,11 @@ public Mat3() {
             0.0f, 1.0f, 0.0f,
             0.0f, 0.0f, 1.0f};
     }
+
+    public float get(int i, int j) {
+        if (i < 0 || i >= 3 || j >= 3 || j < 0) {
+            throw new IllegalArgumentException("Invalid indices");
```

Comment:
Please use IndexOutOfBoundsException instead of IllegalArgumentException, as this is the most appropriate option in the get(i: int, j: int) method. Mat4 also follows this option to maintain overall code consistency.